src/test/resources/integrationtests/sso/testSSOFieldLauncherService.feature

```diff
@@ -0,0 +1,42 @@
+#Author: andrew.johnys@ext.ons.gov.uk
+#Keywords Summary : CCS CENSUS COVERAGE SURVEY
+#Feature: Test SSO to EQ using Field Launcher Service
+#Scenario: Authentication of SSO
+#Scenario: Display message to the field officer when the response has already submitted
+#Scenario: Field Officer is already authenticated
+#Scenario: Display message to the field officer when the case id is invalid
+#
+#The following scenario is also covered by the scenario 'Authentication of SSO'
+#Scenario: Display SSO Authentication UI
+## (Comments)
+Feature: Test SSO to EQ using Field Launcher Service
+  I want to verify that I can access the EQ using SSO for the Field Launcher Service
+
+  @SetUpFieldServiceTests @TearDown
+  Scenario: Authentication of SSO
+    Given I am a field officer and I have access to a device with SSO
+    And I click on the job URL⁠ in the chrome browser
+    And a field proxy authentication UI is displayed on the screen
```

Comment:
I notice that the next test doesn't do this step, which makes me think that perhaps it should be deleted and it's checking merged into the enter-sso-credentials step.